src/utilities/svgs.js

```diff
@@ -33,14 +33,15 @@ import LoadingToggleIconPath from 'assets/icons/loadingToggle.svg';
 import ManageFiltersIconPath from 'assets/icons/manageFilters.svg';
 import PhysicalDeviceIconPath from 'assets/icons/physicalDevice.svg';
 import PlusIconPath from 'assets/icons/plus.svg';
+import QuestionMarkIconPath from 'assets/icons/questionMark.svg';
 import RadioSelectedIconPath from 'assets/icons/radioSelected.svg';
 import RadioUnselectedIconPath from 'assets/icons/radioUnselected.svg';
 import ReconfigureIconPath from 'assets/icons/reconfigure.svg';
 import RefreshIconPath from 'assets/icons/refresh.svg';
 import RuleDisabledIconPath from 'assets/icons/ruleDisabled.svg';
 import RuleEnabledIconPath from 'assets/icons/ruleEnabled.svg';
-import SettingsIconPath from 'assets/icons/settings.svg';
 import SIMManagementIconPath from 'assets/icons/SIMManagement.svg';
+import SettingsIconPath from 'assets/icons/settings.svg';
```

Comment:
Alphabetically... se comes before si... so the old location would be correct.

Reply:
This is probably coming from how VS code line sort is handling upper vs lowercase. Best to leave it with the default behavior or else it will be undone later on anyway.